src/commands/local.ts

```diff
@@ -652,10 +652,23 @@ async function startServerHttp(
     processForClasses: Map<string, ClassProcess>,
 ): Promise<http.Server> {
     const app = express();
+    const __filename = url.fileURLToPath(import.meta.url);
+    const __dirname = path.dirname(__filename);
     app.use(cors());
     app.use(bodyParser.raw({ type: () => true, limit: "6mb" }));
     app.use(genezioRequestParser);
 
+    // serve test interface built folder on localhost
+    const builtFolder = path.resolve(
+        __dirname,
+        "../../../node_modules/genezio-test-interface-component/dist/build",
+    );
+    app.use(express.static(builtFolder));
+    app.get(`/test-interface/local/`, (req, res) => {
```

Comment:
Bogdan Ripa had some ideas on what path we should use. Maybe talk to him about this. This path is too long and does not mean anything to the user.

```suggestion
    app.get(`something else`, (req, res) => {
```

Reply:
we will go with `/explore`